examples/graceful_shutdown.py

```diff
@@ -25,6 +25,16 @@ async def send_json(
         await self.send(json_dumps(data), id=id, event=event, retry=retry)
 
 
+async def send_event(stream, data, event_id):
+    try:
+        await stream.send_json(data, id=str(event_id))
+    except Exception as exc:
+        print(
+            f"Got an exception with event {event_id}, "
+            f"cause {type(exc).__name__}: {exc}"
+        )
```

Comment:
Should probably be an actual logging call here.

```suggestion
        logging.exception("Exception when sending event: %s", event_id)
```